### PR TITLE
Develop v2

### PR DESCRIPTION
**Pull request by JinweiZhu**
Most of the chatroom part has not been changed since last review.

### CODE REVIEW

File: .github/actions/deploy/action.yml [hunk 1]
```diff
@@ -32,5 +32,5 @@ runs:
       working-directory: voluncheer
       run: |
         eb init ${{ inputs.app-name }} --region ${{ inputs.region }} --platform ${{ inputs.platform }}
-        eb deploy ${{ inputs.app-name }}-${{ github.ref_name }} --label ${{ inputs.app-name}}_${{ github.ref_name }}_${{ steps.timestamp.outputs.TIMESTAMP }}
+        eb deploy ${{ inputs.app-name }}-${{ github.ref_name }}-v2 --label ${{ inputs.app-name}}_${{ github.ref_name }}_${{ steps.timestamp.outputs.TIMESTAMP }}
```
Comment (joeleu) [inline .github/actions/deploy/action.yml:35]: So we are deploying to main-v2 and develop-v2?

Did you give the right permission to the two deployment roles?

**Review by joeleu — CHANGES_REQUESTED** [review]
Please add the tests back too (I had to remove them earlier).

I thought we needed the Procfile too?

File: .github/actions/deploy/action.yml (hunk 1 above)
Comment (JinweiZhu) [reply .github/actions/deploy/action.yml:35]: I did last time for dev. I am doing the same for main.

Comment (joeleu) [reply .github/actions/deploy/action.yml:35]: Did you give the right permission to the two deployment roles?

**Comment by joeleu** [comment]
> Please add the tests back too (I had to remove them earlier).
> 
> I thought we needed the Procfile too?

Whoops, I see the Procfile. Just need the tests

File: voluncheer/.ebextensions/01_django.config [hunk 2]
```diff
@@ -17,6 +17,9 @@ container_commands:
     command: "echo '*/10 * * * * root source /var/app/venv/*/bin/activate && python3 manage.py archive_opportunities' > /etc/cron.d/archive_opportunities"
   07_set_archive_permissions:
     command: "chmod 644 /etc/cron.d/archive_opportunities"
+  08_wsgipass:
+    command: 'echo "WSGIPassAuthorization On" >> ../wsgi.conf'
```
Comment (joeleu) [inline voluncheer/.ebextensions/01_django.config:21]: Should we just put this file in the code instead of creating it every time we deploy?

**Review by joeleu — CHANGES_REQUESTED** [review]

File: .github/actions/deploy/action.yml (hunk 1 above)
Comment (JinweiZhu) [reply .github/actions/deploy/action.yml:35]: I think I did.

File: voluncheer/.ebextensions/01_django.config (hunk 2 above)
Comment (JinweiZhu) [reply voluncheer/.ebextensions/01_django.config]: Doesn't seem necessary anymore.

**Comment by JinweiZhu** [comment]
> > Please add the tests back too (I had to remove them earlier).
> > I thought we needed the Procfile too?
> 
> Whoops, I see the Procfile. Just need the tests

The tests are already merged in the last PR of chatroom app. This new PR is merely configurations.

**Pull request merged**